build_tools/travis/install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -e
+
+echo 'List files from cached directories'
+echo 'pip:'
+ls $HOME/.cache/pip
+
+# Deactivate the travis-provided virtual environment and setup a
+# conda-based environment instead
+deactivate
+
+# Use the miniconda installer for setup of conda itself
+pushd .
+cd
+mkdir -p download
+cd download
+if [[ ! -d /home/travis/miniconda3 ]]
```

Comment:
I'd prefer to explicitly have it set up conda, so we have a unified  environment across dev + CI.